src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/util/LogFileHelper.java

```diff
@@ -68,4 +70,15 @@ public static void createFileAndWriteData(Path tempDirectoryPath, String fileNam
             addDataToFile(messageBytes, file.toPath());
         }
     }
+
+    public static LogFile createTempFileAndWriteDataAndReturnFile(Path tempDirectoryPath, String fileNamePrefix)
```

Comment:
Why do we need this? - This is doing the same as the one above. Just change the one above to return the file it should not break anything since existing callers will just ignore the returned result